GeneralizedMorphismsForCAP/gap/GeneralizedMorphismCategory.gi

```diff
@@ -42,8 +42,13 @@ InstallMethod( AssociatedMorphism,
                [ IsGeneralizedMorphism ],
                
   function( generalized_morphism )
+    local associated_morphism;
     
-    return DomainAssociatedMorphismCodomainTriple( generalized_morphism )[2];
+    associated_morphism := DomainAssociatedMorphismCodomainTriple( generalized_morphism )[2];
+    
+    INSTALL_TODO_LIST_FOR_GENERALIZED_MORPHISMS( associated_morphism, generalized_morphism );
```

Comment:
This is mathematically wrong.
The properties 
`IsMonomorphism,` `IsEpimorphism,` `IsIsomorphism` cannot be mirrored between
a generalized morphism and its associated morphism.
Such a propagation would make sense in the constructors
`AsGeneralizedMorphism`.

The property
`IsZero`
doesn't make sense for generalized morphisms.